.github/workflows/cibuildwheel.yml

```
@@ -15,8 +15,18 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-18.04, windows-latest, macos-latest]
+        cibw_archs: ["auto"]
+        include:
+          - os: ubuntu-18.04
+            cibw_archs: "aarch64"
 
     steps:
+      - name: Set up QEMU
+        if: runner.os == 'Linux'
```

```suggestion
        if: matrix.cibw_archs == 'aarch64'
```

This way you only activate this for the one run it's needed for.